library/std/src/Std/Diagnostics.qs

```diff
@@ -368,6 +368,46 @@ operation StopCountingQubits() : Int {
     body intrinsic;
 }
 
+/// # Summary
+/// Configures Pauli noise for simulation.
+///
+/// # Description
+/// This function configures Pauli noise for simulation. Parameters represent
+/// probabilities of applying X, Y, and Z gates and must add up to at most 1.0.
+/// Noise is applied after each gate and before each measurement in the simulator
+/// backend. Decompositions may affect the number of times noise is applied.
+/// Use 0.0 for all parameters to simulate without noise.
+///
+/// # Input
+/// ## px
+/// Probability of applying X gate.
+/// ## py
+/// Probability of applying Y gate.
+/// ## pz
+/// Probability of applying Z gate.
+@Config(Unrestricted)
```

Comment:
AFAIK, these shouldn't be limited to `Unrestricted`, because we can still run the simulator in `Base` or `Adaptive` configuration, and applying noise would still be a valid scenario.

Reply:
Yeah, since these do not return a value other than `Unit` they can be safely treated as no-op in QIR codegen (they will never affect control flow) and as such can have the config restriction removed. This is similar to how `DumpMachine()` and `DumpRegister()` are treated.

Reply:
Removed Unrestricted restriction :)
Original thought was that this only valid for simulator, so we would want to have this in unrestricted only. But we do simulate other profile.